### PR TITLE
Add spawn command and improve world monitoring

### LINKED ISSUE #27: Recreate spawn factory monitoring functionality
- [ ] create entity
- [ ] create command 
- [ ] create script for monitoring the factories

### PR DESCRIPTION
**Pull request by Natty-Boh**
closes #27

### CODE REVIEW

File: src/entity/LandAndFacilities.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Column, Entity, PrimaryGeneratedColumn, Index } from "typeorm";
 @Index(["country", "timestamp"])
 export class LandAndFacilities {
     @PrimaryGeneratedColumn()
-    id!: number;
```
Comment (TotallyNot) [inline src/entity/LandAndFacilities.ts:7]: this isn't nice...

File: src/handler/global/spawns.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                "Number of hours needs to be greater than zero!"
+            );
+        }
+        const hoursInMs = 60000 * 60 * hours;
+        const countryMap = await getCountries();
+        const facs = await LandAndFacilitiesRepository.getSpawnFactories(
+            new Date(Date.now() - hoursInMs)
+        );
+
+        const allies = facs.reduce<FacQueryRow[]>((acc, e) => {
```
Comment (TotallyNot) [inline src/handler/global/spawns.ts:41]: specifying the type here isn't necessary, I think

Comment (TotallyNot) [reply src/handler/global/spawns.ts:41]: actually, what are you even doing here. Why not use `filter`

Comment (Natty-Boh) [reply src/handler/global/spawns.ts:41]: the type isnt necessary but vs code yells if its not there so i added it

Comment (Natty-Boh) [reply src/handler/global/spawns.ts:41]: as to why i used reduce... uh i changed my approach and blindly reused code without thinking about it whoops will change lol

**Review by TotallyNot — APPROVED** [review]

**Pull request merged**